src/components/sphere/Sphere.tsx

```diff
@@ -9,16 +9,7 @@ import { SphereProps } from "../../types/SphereProps";
 const Sphere: React.FC<SphereProps> = (props: SphereProps) => {
 	return (
```

Comment:
Add this parsing function which you'll use below (see seperate comment)
```suggestion

function parseSizeProp(size:string):string{
  size=size.startsWith('sphere-')?size.slice(7):size
  if(!['micro','xsmall','small','medium','large','big','xlarge'].contains(size)){
    console.warn("Not a recognized size for Spheres: "+size);
    return 'large'
  }
  return size
}
return (
```